wp-cli.php

```diff
@@ -387,16 +391,89 @@ public function ip_list( $args, $assoc_args ) {
 	public function ip_add( $args, $assoc_args ) {
 		$this->setup( $args, $assoc_args );
 
+		/**
+		 * The input arguments will can be of the form:
+		 * wp rsa ip-add 8.8.8.8=Google 9.9.9.9 1.1.1.1=Cloudflare.
+		 *
+		 * Some input IP addresses may be provided with a label while
+		 * other might not.
+		 *
+		 * We will normalise the input as:
+		 * array(
+		 *     array(
+		 *         'key' => '8.8.8.8',
+		 *         'label' => 'Google'
+		 *     ),
+		 *     array(
+		 *         'key' => '9.9.9.9',
+		 *         'label' => ''
+		 *     ),
+		 *      array(
+		 *         'key' => '1.1.1.1',
+		 *         'label' => 'Cloudflare'
+		 *     ),
+		 * )
+		 */
+		$ips_and_labels_array = array();
+		foreach ( $args as $index => $item ) {
+			$fragments = explode( '=', $item );
+			/**
+			 * If the IP doesn't have a corressponding label,
+			 * then set label to '[null]x', where 'x' is an
+			 * integer.
+			 */
+			if ( ! isset( $fragments[1] ) ) {
+				$fragments[1] = '[null]:' . $index++;
+			}
```

Comment:
Can you please explain why do we need this? The `Restricted_Site_Access::add_ips` supports labels already.

Reply:
@dinhtungdu I modified the `ip-add` command to make it work along the same lines as the GUI.
Currently, the GUI allows you to make labels optional, but that isn't the case with CLI (see #153 )

This PR allows to add multiple IPs with optional labels, meaning some IPs can have labels while others can be added without a label.
Due to the constraint in #153, I have set a placeholder label as `[null]:x` (I know this can be better named), which is used to set an empty label.